### ISSUE TITLE
Several Issues

### ISSUE THREAD

**Issue by DominusMaximus** [demilich1/metastone#280]
- Adjusting available mana in Sandbox Mode isn't working.

- When casting a Firelands Portal on an enemy minion with a Djinni of Zephyrs on my side of the board, the Portal is copied and cast on the Djinni.

- With 10 cards in hand, Wild Growth is no longer generating an Excess Mana card.

- "Dopp _el_ gangster" should be spelled "Dopp _le_ gangster".

- Brass Knuckles should belong to Warriors.

- The Bomb Squad was captured by several Bluegill Warriors while attempting to operate as undercover _2/1 Murlocs for 2 Mana_. Apparently their subpar mrgls gave them away.

- Arghh, how many Bloody Cultists must I pay before me weapons are sharpened through.

**Comment by webadict** [comment]
Omg... How did I break everything already?

Alright, working on it.

Also, it is Doppelgangster. It derives from the word doppelganger, which, while looking weird, is correct. Blame the Deutschen background of the word für das.

**Comment by webadict** [comment]
1. Fixed the mana issue. I made a mistake while creating vastly superior bug tracking.
2. Uh... Wow, this one is actually a two parter. So, apparently, this is currently incorrectly setup. Right now, it's working with the TragetAcquisitionPhase, which means that it has the potential to copy spells that were not targeting correctly. This WASN'T the issue, though. The issue is that the source player and the target player were being set to the same person. But guess what? I'm amazing, so it works perfectly now. I've moved the trigger to the AfterSpellCastedTrigger... But I also had to move a few things around so that I could actually do that. As far as I'm aware, I've solved everything. Which means I've inevitably broken something else. And two other things.
3. I can't actually get this one to happen...? Perhaps you had a Gadgetzhan Auctioneer that drew you a card before you received Excess Mana (Which would be the correct ordering.) That's my only guess.
4. Dopplegangster is a dumb name that makes no sense. Someone should be fired for that. The word is DOPPELGANGER! Who do I blame for this?!?
5. Fixed. Gave the Warriors more weapons.
6. ... Uh... I actually don't know how that happened. Oh well, fixed.
7. 2? I think?!? I don't even know, man! But apparently, there was some sort of issue where it only counted minions that were currently being summoned. Which is fixed now. The REAL issue was that it wasn't targeting a weapon when it was buffing. Whooooooops.

TL;DR: Fixed, because I'm amazing.

**Comment by DominusMaximus** [comment]
- I can't reproduce #3 either. Not sure what happened there. Sorry about that. 

- I remember back in Magic: the Gathering there were several cards with the word Doppelganger. I thought Blizzard had decided to simplify the spelling, but today I found a thread where Mike Donais says it was just a mistake and that the spelling will be corrected. I guess that means we blame Ben Brode?

That was some quick fixin. Nice.

**Comment by webadict** [comment]
To be fair, it's easy to fix when you know what to look for. Keep finding those bugs to crush.
